Tryouts/Core/MorganStanley.ComposeUI.Tryouts.Interfaces/IoCNames.cs

```diff
@@ -0,0 +1,18 @@
+﻿/// ********************************************************************************************************
+///
+/// Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+/// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+/// See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+/// Unless required by applicable law or agreed to in writing, software distributed under the License
+/// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and limitations under the License.
+/// 
+/// ********************************************************************************************************
+
+namespace MorganStanley.ComposeUI.Playground.Interfaces
+{
+    public static class IoCNames
+    {
+        public const string TimeServiceName = "TimeService";   
```

Comment:
Good luck changing the name and then loading a plugin that was built against the older version ;) In contrast, just injecting an `ITimeService` type will only break if the new version really has a breaking change in method signatures etc.

Reply:
This is just a preliminary test. There is a way to control the dll dependencies in .NET 6.0 and higher which will have employ at some point.